src/main/java/org/sonarlint/intellij/ui/SonarLintWalkthroughToolWindow.java

```diff
@@ -0,0 +1,482 @@
+import com.intellij.util.ui.UIUtil;
+import java.awt.BorderLayout;
+import java.awt.CardLayout;
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+import java.awt.Font;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.util.Objects;
+import javax.swing.BorderFactory;
+import javax.swing.ImageIcon;
+import javax.swing.JButton;
+import javax.swing.JEditorPane;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.SwingConstants;
+import javax.swing.event.HyperlinkEvent;
+import org.jetbrains.annotations.NotNull;
+import org.sonarlint.intellij.config.global.SonarLintGlobalConfigurable;
+import org.sonarlint.intellij.config.project.SonarLintProjectConfigurable;
+
+import static org.sonarlint.intellij.documentation.SonarLintDocumentation.Intellij.RULE_SECTION_LINK;
+import static org.sonarlint.intellij.telemetry.LinkTelemetry.AI_FIX_SUGGESTIONS_PAGE;
+import static org.sonarlint.intellij.telemetry.LinkTelemetry.BASE_DOCS_PAGE;
+import static org.sonarlint.intellij.telemetry.LinkTelemetry.RULE_SELECTION_PAGE;
+
+public class SonarLintWalkthroughToolWindow implements ToolWindowFactory, DumbAware {
+
+  protected static final String SONARQUBE_FOR_IDE = "SonarQube for IDE";
+  protected static final String PAGE_2 = "Page 2";
+  protected static final String PAGE_3 = "Page 3";
+  protected static final String FONT = "Arial";
+  protected static final String EDITOR_PANE_TYPE = "text/html";
+  protected static final String PREVIOUS = "Previous";
+
+  @Override
+  public void createToolWindowContent(Project project, ToolWindow toolWindow) {
+    var font = UIUtil.getLabelFont();
+    var mainPanel = new JPanel(new CardLayout());
+    mainPanel.setPreferredSize(new Dimension(70, 100));
+
+    var welcomePage = new JPanel(new BorderLayout());
+    var learnAsYouCodePage = new JPanel(new BorderLayout());
+    var connectWithYourTeamPage = new JPanel(new BorderLayout());
+    var reachOutToUsPage = new JPanel(new BorderLayout());
+
+    var icon = new ImageIcon(Objects.requireNonNull(getClass().getResource("/images/sonarqube-for-ide-mark.png")));
+    var welcomeImageLabel = new JLabel(icon);
+    var learnAsYouCodeImageLabel = new JLabel(icon);
+    var connectWithYourTeamImageLabel = new JLabel(icon);
+    var reachOutToUsImageLabel = new JLabel(icon);
+
+    var welcomeStepLabel = new JLabel("Step 1/4", SwingConstants.LEFT);
+    welcomeStepLabel.setFont(new Font(FONT, Font.PLAIN, 14));
+
+    var titleLabel = new JLabel("Get started", SwingConstants.LEFT);
+    titleLabel.setFont(new Font(FONT, Font.BOLD, 16));
+    var welcomePageText = createWelcomePageText(font, project);
+
+    var welcomePageScrollPane = new JScrollPane(welcomePageText);
+    welcomePageScrollPane.setBorder(null);
+    welcomePageScrollPane.setPreferredSize(new Dimension(70, 100));
+
+    var welcomePageNextButton = new JButton("Next: Learn as you code");
```

Comment:
```suggestion
    var welcomePageNextButton = new JButton("Next: Learn as You Code");
```

Reply:
Button labels should have [title-case capitalization](https://plugins.jetbrains.com/docs/intellij/button.html#use-title-capitalization)